src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/CommandMode.java

```diff
@@ -114,11 +146,12 @@ public EndDescriptor() {
         private String source;
 
         @DataBoundConstructor
-        public UploadReportsView(String reportFiles, String reportsFolders, boolean hasMoreRequests, String source) {
-            super(CommandModes.UploadReports);
+        public UploadReportsView(String reportFiles, String reportsFolders,
+                                 String source, String buildSessionId, String additionalArguments) {
+            super(CommandModes.UploadReports, buildSessionId, additionalArguments);
             this.reportFiles = reportFiles;
             this.reportsFolders = reportsFolders;
-            this.hasMoreRequests = hasMoreRequests;
+            this.hasMoreRequests = true;
```

Comment:
A. Is there a reason why the 'hasMoreRequests' is now 'true'?
B. If so, does anyone can override it? If not, can we remove the field?